bot/exts/filters/token_remover.py

```diff
@@ -18,10 +18,10 @@
     "Censored a seemingly valid token sent by {author} (`{author_id}`) in {channel}, "
     "token was `{user_id}.{timestamp}.{hmac}`"
 )
-DECODED_LOG_MESSAGE = "The token user_id decodes into {user_id}."
-USER_TOKEN_MESSAGE = (
+UNKNOWN_USER_LOG_MESSAGE = "The token user_id decodes into {user_id}."
+KNOWN_USER_LOG_MESSAGE = (
     "The token user_id decodes into {user_id}, "
-    "which matches `{user_name}` and means this is a valid USER token."
+    "which matches `{user_name}` and means this is a valid {kind} token."
 )
```

Comment:
The first half of these messages is identical. Would it complicate things to try to remove this redundancy?

Reply:
I'm going to reword how these messages flow, there's no reason for them to be similar (which is why they are separate). Something more like existing commands, I think